charts/incubator/notifiarr/values.yaml

```diff
@@ -0,0 +1,54 @@
+image:
+  repository: tccr.io/truecharts/notifiarr
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:913c71f5e325dd9af53c9e96f3d5d7d1b2d058a41f8253489320a4c69fbb0a44
+
+notifiarr:
+  upstreams:
+    - 172.16.0.0/16
+    - upstream1
+    - upstream2
+    - upstream3
+    - upstream4
+  credentials:
+    - username
+    - password
```

Comment:
Why is that a list?!

credentials:
  username:
  password:
  
{{ printf "%s:%s" .Values.noti.cred.username .Values.notifi.credent.password }}